### ISSUE TITLE
Example is missing .env file

### ISSUE THREAD

**Issue by tbecker** [liodali/osm_flutter#291]
For testing, the .env file is missing. Please add in the asset folder in the example

**Comment by liodali** [comment]
yeah i know that you can do it in your local because my env file contain api keys that shouldnt be exposed 😅😅 sorry 
so jist create empty .env 
and if your own server need auth key put the key there

**Comment by tbecker** [comment]
Please add descripiton or an empty .env in the description.
If I add an empty .env I get Unhandled Exception: Instance of 'FileNotFoundError' although I have the file in asset and in pubspec.yaml

**Comment by liodali** [comment]
can you provide me more information 
did you fork the project or just the example of customTile ?

**Comment by tbecker** [comment]
I copied the example file directory to local and tried to run.

**Comment by tbecker** [comment]
Sorry, it was my mistake. First, the .env file was in the wrong directory and after I moved it there, there is still a complained about empty contents. So I added some dummy text in there. This Issue can be closed, but I still recommend that the need for an non-empty env is either included as comment in the source code or a dummy file is created for the sake of getting to a good fast first use experience.
